x/tss/keeper/msg_server.go

```diff
@@ -208,15 +204,12 @@ func (s msgServer) StartKeygen(c context.Context, req *types.StartKeygenRequest)
 	telemetry.SetGauge(float32(minKeygenThreshold.Numerator*100/minKeygenThreshold.Denominator), types.ModuleName, "minimum", "keygen", "threshold")
 
 	// metrics for keygen participation
-	ts := ctx.BlockTime().Unix()
 	for _, validator := range snapshot.Validators {
-		telemetry.SetGaugeWithLabels(
-			[]string{types.ModuleName, "keygen", "participation"},
-			float32(validator.ShareCount),
+		telemetry.SetGaugeWithLabels([]string{types.ModuleName, "keygen", "participation"}, 1,
```

Comment:
what's the meaning of 1 here?

Reply:
Just to have a constant value for the gauge, given that it does not make much sense to set a value for the keygen/sig metrics. I guess it might as well be simply 0